inspirehep/search/factories/search.py

```diff
@@ -28,8 +28,8 @@ def inspire_search_factory(self, search):
     try:
         search = search.query_from_iq(query_string)
     except SyntaxError:
-        current_app.logger.debug(
-            "Failed parsing query: '%s'", request.values.get("q", ""), exc_info=True
+        current_app.logger.exception(
```

Comment:
logging exception + raising it. Is it catched somewhere else?

Reply:
we can just change it to normal logging as the others :)

Reply:
That's great news! :+1: